src/rayTracing.py

```diff
@@ -0,0 +1,127 @@
+# /*************************************************************************
+#  * Copyright (C) 2012-2014 Tavian Barnes <tavianator@tavianator.com>     *
+#  * https://github.com/tavianator/dimension                               *
+#  * Copyright (C) 2020 Pero Prebeg <Pero.Prebeg@fsb.hr>                   *
+#  *                                                                       *
+#  * The content of this file is part of The Dimension Library. The        *
+#  * original C Code of The Dimensions Library has been translated to      *
+#  * Python code, further accomodations have been made to emphasize the    *
+#  * purpose of d3v.                                                       *
+#  *                                                                       *
+#  * The Dimension Library is free software; you can redistribute it and/  *
+#  * or modify it under the terms of the GNU Lesser General Public License *
+#  * as published by the Free Software Foundation; either version 3 of the *
+#  * License, or (at your option) any later version.                       *
+#  *                                                                       *
+#  * The Dimension Library is distributed in the hope that it will be      *
+#  * useful, but WITHOUT ANY WARRANTY; without even the implied warranty   *
+#  * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU  *
+#  * Lesser General Public License for more details.                       *
+#  *                                                                       *
+#  * You should have received a copy of the GNU Lesser General Public      *
+#  * License along with this program.  If not, see                         *
+#  * <http://www.gnu.org/licenses/>.                                       *
+#  *************************************************************************/
+
+import math
+
+from bounds import BBox
+from pyVector3 import pyvector3
+
+
+class dmnsn_ray:
```

Comment:
would you ps explain what is the purpose of this class?

What does `dmnsn` mean?

Reply:
Replaced by custom class with clearer name

Reply:
I do appreciate you allocating some time to answer my questions.

However, there is a problem, I see only your answers and not the changes you are referring to.  The last commit is dated Oct. 8th.

Reply:
All changes are currently commited in another branch. I will update the commits to this branch soon, probably today afternoon. I will give you a note then.